util/src/main/java/org/killbill/billing/util/glue/KillBillShiroModule.java

```diff
@@ -145,6 +144,8 @@ protected void bindSecurityManager(final AnnotatedBindingBuilder<? super Securit
         //super.bindSecurityManager(bind);
         bind.toInstance(defaultSecurityManager);
 
+        // FIXME-1615 : Intellij report there' duplicate line like this, which is in KillBillShiroWebModule line 86.
```

Comment:
`KillBillShiroModule` and `KillBillShiroWebModule` are very similar, but serve quite a different purpose. Not sure it's worth abstracting it (e.g. through a base class) though.